acquia_lift.admin.wizard.inc

```diff
@@ -1212,6 +1260,15 @@ function acquia_lift_personalize_campaign_wizard_goals_ajax_delta($form, &$form_
 }
 
 /**
+ * Ajax handler for updating an existing goal
+ */
+function acquia_lift_personalize_campaign_wizard_goals_ajax_existing($form, &$form_state) {
+  drupal_get_messages();
```

Comment:
Any little messages like "There is a new drupal security version", etc. will get shown within the context of the returned ajax - so basically they pop up in the middle of the page.  Since they would have seen any important messages on the initial page load, I'm suppressing them in these callbacks.